src/main/java/com/linkedin/kafka/clients/largemessage/LargeMessage.java

```diff
@@ -39,12 +40,12 @@ public synchronized long bufferedSizeInBytes() {
   }
 
   public synchronized SegmentAddResult addSegment(LargeMessageSegment segment, long offset) {
```

Comment:
+1 on starting offset. i'd leave the offset in case we ever want to support out-of-order arrival of segments